pkgs/applications/editors/vim/plugins/generated.nix

```
@@ -11146,6 +11146,18 @@ final: prev:
     meta.homepage = "https://github.com/camspiers/snap/";
   };
 
+  snacks-nvim = buildVimPlugin {
+    pname = "snacks.nvim";
+    version = "2024-11-07";
+    src = fetchFromGitHub {
+      owner = "folke";
+      repo = "snacks.nvim";
+      rev = "334895c5bb2ed04f65800abaeb91ccb0487b0f1f";
+      sha256 = "uKAdjRLUtKi6FlE0IHGlnW0Bv2hQhoTRiarvwyU0YaE="
```

```suggestion
      sha256 = "uKAdjRLUtKi6FlE0IHGlnW0Bv2hQhoTRiarvwyU0YaE=";
```
Looks like this got dropped somehow

---

Fixed.